rust/private/common.bzl

```diff
@@ -25,7 +25,11 @@ In the Bazel lingo, `rust_common` gives the access to the Rust Sandwich API.
 
 load(":providers.bzl", "CrateInfo", "DepInfo")
 
+def _create_crate_info(**kwargs):
+    return CrateInfo(**kwargs)
+
 rust_common = struct(
+    create_crate_info = _create_crate_info,
```

Comment:
I know it's not exported to https://bazelbuild.github.io/rules_rust/, but could you also add a doc saying that create_crate_info should be used for construction, and crate_info for accessing?

Reply:
Like, a docstring? (like the one I added?)

Reply:
Oh right, yeah that works. Nit: rename "macro" to "function" in the docstring

Reply:
Is the terminology not "macro"? But updated.